### ISSUE TITLE
Enhancements for SaaS use-case

### ISSUE THREAD

**Issue by whalehub** [Ylianst/MeshCentral#87]
As I mentioned in another issue, I plan on offering hosted instances of MeshCentral server as SaaS to SMBs, MSPs and self-employed IT techs in exchange for a monthly fee. One thing that many other enterprise remote control tools do is limit the amount of concurrent users and sessions and/or the total number of machines based on the tier that the customer pays for. It's currently not possible to place such limitations on individual users or organisations (domains). It would be easier to offer different tiers of SaaS for MeshCentral if it was possible to restrict the number of concurrent users/sessions and the total number of connected machines.

**Comment by Ylianst** [comment]
Hi. Yes, good idea. I will add that. So, in the config.json, for each domain... I will have maximum number of accounts, usersessions, device and devicesessions. I also wanted to add customization per domain of the login screen text and image at some point.

**Comment by Ylianst** [comment]
Just published MeshCentral 0.2.7-w. In the domains section, you can now do this:
```
  "domains": {
    "": {
      "Limits": {
        "MaxUserAccounts": 100,
        "MaxUserSessions": 100,
        "MaxAgentSessions": 100
      }
    }
```
Please try it out and let me know if it works for you. Agents that connect above the limit will be have their connection on hold by the server, but will not be active if a spot free's up. Users will get a error that there are too many sessions and can retry until a spot free's up. Feedback appreciated.

**Comment by whalehub** [comment]
@Ylianst Sorry about the delay, had a busy week. I tested the three limits and can confirm that they're all working perfectly. Thanks for this implementing this feature!
